components/log-viewer-webui/server/src/routes/query.js

```diff
@@ -1,5 +1,9 @@
 // eslint-disable-next-line no-magic-numbers
 const EXTRACT_IR_TARGET_UNCOMPRESSED_SIZE = 128 * 1024 * 1024;
+const EXTRACT_JSON_TARGET_CHUNK_SIZE = 100000;
+
+import {QUERY_JOB_TYPE} from "../DbManager.js";
+
 
```

Comment:
```suggestion
import {QUERY_JOB_TYPE} from "../DbManager.js";


// eslint-disable-next-line no-magic-numbers
const EXTRACT_IR_TARGET_UNCOMPRESSED_SIZE = 128 * 1024 * 1024;
const EXTRACT_JSON_TARGET_CHUNK_SIZE = 100000;

```

Reply:
We can write the `100 * 1000` if that improves readability. (Note we also need the eslint suppression line.)